### DISCUSSION TITLE
LEGO ev3dev sd image 2.0.0 reports Pybricks MicroPython v1.11

### DISCUSSION THREAD

**Discussion by BertLindeman** [pybricks/support#1236] [Q&A] [answered]
Flashed ```ev3micropythonv200sdcardimage.zip``` from LEGO and booted the EV3 from it.

No changes applied and expected to see pybricks-micropython version 2.0, but I see:

```
pybricks-micropython
Pybricks MicroPython v1.11 on 2020-05-06; linux version
Use Ctrl-D to exit, Ctrl-E for paste mode
>>>
```

To be able to update I needed to change ```/etc/apt/sources.list``` to
```
deb http://archive.debian.org/debian/ stretch main contrib non-free

deb http://archive.debian.org/debian/ stretch-proposed-updates main contrib non-free

deb http://archive.ev3dev.org/debian stretch main
```

```apt update && apt upgrade -V``` did not install another pybricks-micropython, so no change there.

What should I do (have done?) to indeed have pybricks-micropython 2.0?

**Comment by dlech** [comment, marked as answer]
You don't have to do anything. We just didn't have the version string customized back then. Pybricks 2.0 was based on MicroPython 1.11.

**Reply by BertLindeman** [reply]
Aaaaah.

Thank you David.
